deploy/deploy.sh

```diff
@@ -1,34 +1,20 @@
 #! /bin/bash
 
-# Modify THE WANDB_ENTITY value in `environment.sh` (line 16) before running this script.
+# Modify DH_DEPLOY_KEY value in `environment.sh` (line 19) before running this script.
```

Comment:
Consider being more explicit in naming your constants. DOCKER_HUB_DEPLOY_KEY is more easily parsed than: looking at DH_DEPLOY_KEY, looking through the code to see where it's used, and **_then_** realizing "Oh, this is for the Docker Hub Key."

Reply:
Refactored DH_DEPLOY_KEY -> DOCKER_HUB_DEPLOY_KEY for readability.